app/controllers/api/articles_controller.rb

```diff
@@ -1,15 +1,32 @@
 class Api::ArticlesController < ApplicationController
   def index
-    if params[:article_type]
+    if params[:lat] && params[:long]
+      location = Geocoder.search([params[:lat], params[:long]])
+      raw_list = Article.where(location: location.first.city).sort_by(&:created_at).reverse
+      if raw_list == []
+        raw_list = Article.where(article_type: params[:article_type]).sort_by(&:created_at).reverse
+        render json: {
+          message: 'We found no local articles from Frederiksdal.',
```

Comment:
should this be hard coded?

Reply:
Ugh, definitely not! Pasted over my dynamic one x)